src/components/Products/GridItem/GridItem.js

```diff
@@ -15,110 +16,112 @@ import { getIsFavoriteStatusFromProducts } from 'store/products/selectors';
+        setIsDetailModalVisible(false);
+    };
 
-  const addToCartHandler = () => {
-    dispatch(addToCart(product));
-  };
+    const addToCartHandler = () => {
+        dispatch(addToCart(product));
+    };
 
-  const favoriteClasses = isFavoriteStatus
-    ? 'button-round -favorite'
-    : 'button-round';
+    const favoriteClasses = isFavoriteStatus
+        ? 'button-round -favorite -first-icon'
+        : 'button-round -first-icon';
 
-  const favoriteInfoText = isFavoriteStatus
-    ? 'remove from favorites'
-    : 'add to favorites';
+    const favoriteInfoText = isFavoriteStatus
+        ? 'remove from favorites'
+        : 'add to favorites';
 
-  return (
-    <>
-      <li className="gridItem-container" onClick={redirectToDetailPage}>
-        <div className="gridItem">
-          <div className="gridItem__img-wrap">
-            <img src={imageUrl} />
-          </div>
-          <div className="gridItem__description">
-            <p className="gridItem__text">{name}</p>
-            <p className="gridItem__text">${price / 100}</p>
-          </div>
-          <div className="gridItem__first-icon">
-            <ButtonRound
-              icon={iconFav}
-              onClick={favoriteToggleHandler}
-              info={favoriteInfoText}
-              className={favoriteClasses}
-            />
-          </div>
-          <div className="gridItem__show-icon">
-            <ButtonRound
-              icon={iconDetail}
-              onClick={showDetailModalHandler}
-              info="show details"
-              className="button-round"
-            />
-          </div>
-          <div className="gridItem__show-button">
-            <ButtonLong
-              title="Add to cart"
-              onClick={addToCartHandler}
-              icon={iconBtn}
-            />
-          </div>
-        </div>
-      </li>
-      {isDetailModalVisible && (
-        <Modal
-          element={
-            <ShowProductDetail
-              product={{ imageUrl, name, price, id, description, inStock }}
-            />
-          }
-          onClick={closeDetailsModalHandler}
-        />
-      )}
-    </>
-  );
+    return (
+        <>
+            <li className="grid-item-container">
+                <ButtonRound
+                    className={favoriteClasses}
+                    icon={iconFav}
+                    info={favoriteInfoText}
+                    onClick={favoriteToggleHandler}
+                />
+                <ButtonRound
+                    className="button-round -show-icon"
+                    icon={iconDetail}
+                    info="show details"
+                    onClick={showDetailModalHandler}
+                />
+                <ButtonLong
+                    className="-show-button"
+                    icon={iconBtn}
+                    onClick={addToCartHandler}
+                    title="Add to cart"
+                />
+                <Link className="grid-item" to={`/product/${id}`}>
```

Comment:
On some resolutions the icons are no longer on the image. It would be nicer if the image was always in the background:
https://user-images.githubusercontent.com/2543423/226175491-a1786a54-1f72-4c1b-8405-6d3cc284fc1a.mov